src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/Error/SqlExtendedQueryTagErrorStoreV5.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Client;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag.Error
+{
+    internal class SqlExtendedQueryTagErrorStoreV5 : SqlExtendedQueryTagErrorStoreV4
+    {
+        public SqlExtendedQueryTagErrorStoreV5(
+           SqlConnectionWrapperFactory sqlConnectionWrapperFactory,
+           ILogger<SqlExtendedQueryTagErrorStoreV5> logger) : base(sqlConnectionWrapperFactory, logger)
+        {
+        }
+
+        public override SchemaVersion Version => SchemaVersion.V5;
```

Comment:
Maybe for the future, we should look at making this better. I feels boiler plate code to create a class to just inherit, maybe we can make Version property a collection and multiple versions per class.

Reply:
I agree! This is something that should be fixed sooner than later. As we have increasing number of schemas, this will get harder to manage as well.